src/SFA.DAS.CourseDelivery.Infrastructure.UnitTests/HealthChecks/WhenGettingCourseDirectoryHealthCheck.cs

```diff
@@ -0,0 +1,84 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.CourseDelivery.Domain.Entities;
+using SFA.DAS.CourseDelivery.Domain.Interfaces;
+using SFA.DAS.CourseDelivery.Infrastructure.HealthCheck;
+using SFA.DAS.Testing.AutoFixture;
+using System;
+using System.Collections.Generic;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.CourseDelivery.Infrastructure.UnitTests.HealthChecks
+{
+    public class WhenGettingCourseDirectoryHealthCheck
+    {
+        [Test, MoqAutoData]
+        public async Task Then_The_ImportAudit_Course_Directory_Record_Is_Read_From_The_Repository(
+            [Frozen] Mock<IImportAuditRepository> repository,
+            HealthCheckContext healthCheckContext,
+            CourseDirectoryHealthCheck healthCheck)
+        {
+            //Act
+            await healthCheck.CheckHealthAsync(healthCheckContext);
+
+            //Assert
+            repository.Verify(x => x.GetLastImportByType(ImportType.CourseDirectory), Times.Once);
+        }
+
+        [Test, MoqAutoData]
+        public async Task Then_If_No_Course_Directory_Rows_Are_Loaded_Then_Shows_As_Degraded(
```

Comment:
Test indicates this should be 
`Then_If_No_Course_Directory_Rows_Are_Loaded_Then_Shows_As_Unhealthy`